types/pg/index.d.mts

```diff
@@ -0,0 +1,34 @@
+import type * as pg from "./index.d.ts";
+
+import TypeOverrides = require("./lib/type-overrides");
+
+export type * from "./index.d.ts";
+
+export const defaults: typeof pg.defaults;
+export const Client: typeof pg.Client;
+export const Query: typeof pg.Query;
+export const Pool: typeof pg.Pool;
+export const Connection: typeof pg.Connection;
+export const types: typeof pg.types;
+export const DatabaseError: typeof pg.DatabaseError;
+export const escapeIdentifier: typeof pg.escapeIdentifier;
+export const escapeLiteral: typeof pg.escapeLiteral;
+export const Result: typeof pg.Result;
+
+export { TypeOverrides };
```

Comment:
this export should be in the CJS types too

Reply:
good point, fixed!

Reply:
amazing, thank you!